bundler/lib/bundler/dsl.rb

```diff
@@ -488,6 +489,16 @@ def normalize_source(source)
       end
     end
 
+    def deprecate_legacy_windows_platforms(platforms)
+      dep_platforms = Array(platforms).select {|pl| pl.to_s.match?(/mingw|mswin/) }
```

Comment:
I think we're already passing an Array, so no `Array()` would be needed? Also, maybe `windows_platforms` is better as a variable name?

Reply:
Done